### PR TITLE
treewide: fix typos and indentation

### PR DESCRIPTION
**Pull request by torstehu**

### CODE REVIEW

File: src/hostname/hostnamed.c [hunk 1]
```diff
@@ -212,7 +212,7 @@ static const char* fallback_chassis(void) {
            unreliable enough, so let's not do any additional guesswork
            on top of that.
 
-           See the SMBIOS Specification 4.0 section 7.4.1 for
+           See the SMBIOS Specification 3.0 section 7.4.1 for
```
Comment (zonque) [inline src/hostname/hostnamed.c:215]: Well, the URL to the PDF below references version 3.0, so you should either update the link as well or leave the comment as it is.

Comment (torstehu) [inline src/hostname/hostnamed.c:215]: I'm having a hard time understanding exactly what you are referring to, according to https://www.dmtf.org/standards/smbios the latest publicized release is 3.0.0 which is linked below. Should I change it to 3.0.0?

Comment (zonque) [inline src/hostname/hostnamed.c:215]: Right, there is no newer version. Sorry for the noise.

Comment (torstehu) [inline src/hostname/hostnamed.c:215]: No problem, thanks for the quick review and merge. :)

**Pull request merged**